chapter_optimization/sgd_vn.md

```diff
@@ -248,13 +257,18 @@ it is possible to minimize them in a small number of steps while decreasing the
 Unfortunately this case never really occurs in deep learning and we are left with a much more slowly decreasing rate in practice.
 -->
 
-*dịch đoạn phía trên*
+Phần dưới đây chủ yếu phục vụ để truyền đạt vấn đề này trực quan hơn dù không bắt buộc.
+Chúng ta giới hạn lời giải dưới đây bằng một trong những cách chứng minh đơn giản nhất được trình bày trong :cite:`Nesterov.Vial.2000`.
+Cũng có những lời giải chứng minh cao cấp khác, ví dụ, khi hàm mục tiêu được tối ưu hóa tốt.
+
+:cite: `Hazan.Rakhlin.Bartlett.2008` chỉ ra rằng với các hàm lồi chặt, cụ thể là các hàm có biên dưới là $\mathbf{x}^\top \mathbf{Q} \mathbf{x}$, ta có thể cực tiểu hóa chúng chỉ với số ít bước lặp trong khi giảm tốc độ học chẳng hạn như $\eta(t) = \eta_0/(\beta t + 1)$.
```

Comment:
```suggestion
:cite: `Hazan.Rakhlin.Bartlett.2008` chỉ ra rằng với các hàm lồi chặt, cụ thể là các hàm có cận dưới là $\mathbf{x}^\top \mathbf{Q} \mathbf{x}$, ta có thể tối thiểu hóa chúng chỉ với một số lượng nhỏ bước lặp trong khi giảm tốc độ học theo $\eta(t) = \eta_0/(\beta t + 1)$.
```

Reply:
cận với biên khác nhau đó bạn

Reply:
Từ `cận` với `biên` mình cũng k chắc cái nào là đúng, nhớ đã đọc ở đâu đó là biên. Tuy nhiên có bài báo sử dụng biên như dưới.
http://jst.udn.vn/BBao/2019/5/PDFFull_2019m05d09_10_30_15.pdf

Search qua thì thấy `tối thiểu hóa`  xuất hiện trong kinh tế, `cực tiểu hóa` xuất hiện trong `tối ưu` nói chung (GG).
Ban đầu thì mình cũng định dịch chữ like là `theo` nhưng ở đây chỉ là 1 ví dụ về cách giảm tốc độ học, có lẽ dịch theo đúng nghĩa từ like (ví dụ) thì đúng hơn ?

Reply:
* Mình thấy `cận` có vẻ phổ biến hơn. 
* `minimize` thì giờ mình mới biết hay được dịch là `cực tiểu hóa`, vậy trước giờ toàn dịch sai rồi :)))
* `ví dụ` cũng hợp lý, để mình edit thêm dấu phảy